### PR TITLE
Feature/log2ram bool

### PR DESCRIPTION
**Pull request by crutonjohn**
# Description

Log2ram is useful for Raspberry Pi installations as it prevents unnecessary I/O to SD-based flash storage. However, if a user opts to use USB booting to a more traditional storage solution there may be little to no reason to install log2ram. This PR aims to make log2ram an option.

## Type Of Change

To help us figure out who should review this PR, please put an X in all the areas that this PR affects.

- [ ] Docs
- [X] Installation
- [ ] Performance and Scalability
- [ ] Security
- [ ] Test and/or Release
- [X] User Experience

## Issue Ref (Optional)

None

## Notes

Feel free to contact me on the Discord server @ crutonjohn. I am unfamiliar with the `git commit --signoff` procedure, so if this PR needs to be resubmitted for lack of commit messages or other reasons I understand. I have tested this on my personal cluster with both `true` and `false` options and the behavior was as expected.

### CODE REVIEW

**Review by xunholy — APPROVED** [review]
Everything looks good to me, just need to fix the EOL at EOF where applicable. @RobReus if you could also take a quick look to see if you're happy.

File: ansible/roles/common/defaults/main.yml [hunk 1]
```diff
@@ -19,5 +19,6 @@ common_rpi_poe_fan_temp1: 50000 # Temperature in millicelcius
 common_rpi_poe_fan_temp2: 55000 # Temperature in millicelcius
 common_rpi_poe_fan_temp3: 60000 # Temperature in millicelcius
 
+log2ram_enabled: true # Set to false to skip installing log2ram
```
Comment (RobReus) [inline ansible/roles/common/defaults/main.yml]: Please prefix this with `common_` so its clear that this variable belongs to the `common` role. Ansible has no namespace concept so all variables are global by nature. This is our way to implement a poor man's namespacing.

File: ansible/roles/common/tasks/log2ram.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  args:
+    chdir: '/opt/log2ram'
+  register: log2ram
+  changed_when: '"Reboot to activate log2ram" in log2ram.stdout'
+  notify: reboot hosts
+
+- name: copy log2ram.conf
+  template:
+    src: log2ram.conf.j2
+    dest: /etc/log2ram.conf
```
Comment (RobReus) [inline ansible/roles/common/tasks/log2ram.yml]: Please add an empty line at the end of the file. When using an IDE or more advanced editor, you can often enable this to be done automatically for you.

**Review by RobReus — CHANGES_REQUESTED** [review]
See my 2 (very minor) remarks. Other than that, great work! Thanks a lot for your contribution.

**Review by RobReus — APPROVED** [review]

**Pull request merged**